readthedocs/doc_builder/director.py

```diff
@@ -106,6 +106,20 @@ def setup_vcs(self):
         #
         # self.run_build_job("pre_checkout")
         self.checkout()
+
+        # Output the path for the config file used.
+        # This works as confirmation for us & the user about which file is used,
+        # as well as the fact that *any* config file is used.
+        if self.data.config.source_file:
+            command = self.vcs_environment.run(
+                "cat",
+                # Show user the relative path to the config file
```

Comment:
I think we want to show the relative path starting from the root of the repository. That way we will show something like "cat docs/.readthedocs.yaml". That will tell the user exactly where the yaml is.

If we use the CWD as the base path of the conf file, we won't know the exact path of the file inside the repository

Reply:
Ah right... this makes sense.

Reply:
Pushed a fix 👍